content-src/lib/utils.js

```diff
@@ -1,16 +1,53 @@
+var urlParse = require("url-parse");
```

Comment:
I'll add the ESLint rule in https://github.com/mozilla/activity-streams/pull/218#discussion_r54442320.